indico/modules/events/registration/models/items.py

```diff
@@ -315,11 +317,18 @@ def is_field(cls):
 
     @hybrid_property
     def is_visible(self):
-        return self.is_enabled and not self.is_deleted
+        field_enabled = self.parent.is_enabled and self.is_enabled if self.parent else self.is_enabled
```

Comment:
Or even better:

    return self.is_enabled and not self.is_deleted and (self.parent_id is None or self.parent.is_visible)